packages/cf-style-container/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "css-color-function": "^1.3.0",
+    "fela": "^4.3.2",
```

Comment:
please lock dependencies with yarn.lock. We don't want failed CI builds every time fela upgrades